### PR TITLE
Feature/move from java 8 to 11

### PR DESCRIPTION
**Pull request by DavidDamke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,4 +57,7 @@ You should put the .jar in an extra folder as a *logs* and a *db* folder will be
 
 * Windows 7, 10
 * Linux (tested on Ubuntu 18.04)
```
Comment (Death111) [inline README.md:59]: can you also test the .jar file on windows + linux. When tested also update the readme which Operating systems we tested (Windows 10 + Ubuntu version x)

File: pom.xml [hunk 2]
```diff
@@ -34,8 +34,8 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
```
Comment (Death111) [inline pom.xml:37]: can you also update the github pipeline - we no longer need to compile with java 8 in this case

File: README.md (hunk 1 above)
Comment (DavidDamke) [reply README.md:59]: Works on Windows 10

File: pom.xml (hunk 2 above)
Comment (DavidDamke) [reply pom.xml:37]: Updated the Pipeline. Builds Uploads and Analyzes only with Java 11 now. Code coverage also works with Java 11 now

File: pom.xml [hunk 3]
```diff
@@ -93,64 +119,98 @@
...
-      <plugin>
-        <artifactId>maven-assembly-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>create-keeptime-zip</id>
-            <phase>package</phase>
-            <goals>
-              <goal>single</goal>
-            </goals>
+          <plugin>
```
Comment (Death111) [inline pom.xml:157]: formatting looks of?

File: README.md (hunk 1 above)
Comment (Death111) [reply README.md:69]: on linux only import/export is not working. see https://github.com/doubleSlashde/KeepTime/pull/109

**Pull request merged**